gluten-core/src/main/scala/io/glutenproject/execution/HashAggregateExecBaseTransformer.scala

```diff
@@ -593,6 +593,21 @@ abstract class HashAggregateExecBaseTransformer(
             case other =>
               throw new UnsupportedOperationException(s"not currently supported: $other.")
           }
+        case hllAdapter: HLLVeloxAdapter =>
```

Comment:
gluten-core is a common module for all the backends, can we put this code to velox backend modules?

Reply:
Updated, thanks.

Reply:
+1
Still observed several similar issues.